libc/functions/g/getsid.part-impl.md

```diff
@@ -0,0 +1,38 @@
+# Synopsis 
+`#include <unistd.h>`</br>
+
+`pid_t getsid(pid_t pid);`</br>
+
+## Status
+Partially implemented
+## Conformance
+IEEE Std 1003.1-2017
+## Description
+
+The `getsid()` function shall obtain the process group ID of the process that is the session leader of the process specified by _pid_. If _pid_ is `(pid_t)0`, it specifies the calling process.
+
+## Return value
+
+Upon successful completion, `getsid()` shall return the process group ID of the session leader of the specified process. Otherwise, it shall return `-1` and set _errno_ to indicate the error.
```

Comment:
```suggestion
Upon successful completion, `getsid()` shall return the process group ID of the session leader of the specified process. Otherwise, it shall return `-1` and set `errno` to indicate the error.
```